server/src/main/java/org/jboss/as/server/parsing/VaultXml.java

```diff
@@ -87,7 +87,6 @@ private void parseVault_1_1(final XMLExtendedStreamReader reader, final ModelNod
         final int vaultAttribCount = reader.getAttributeCount();
 
         ModelNode vault = new ModelNode();
-        String code = null;
```

Comment:
Please remove this class from the PR.

Reply:
Thanks for your feedback, this is fixed. I'll wait with a `push --force` until further review is done.